src/styles/blocks/card.scss

```diff
@@ -0,0 +1,44 @@
+.product-container {
+  display: flex;
+  flex-direction: column;
+  max-width: $card-width;
+  padding: 0 14px 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-size: 11px;
+  font-weight: 400;
+
+  &__img {
+    display: block;
+    margin: 29px auto 45px;
+    width: 160px;
+    height: 134px;
+    transition: transform $transition-duration;
+  
+    &:hover {
+      box-sizing: content-box;
+      border: 3px solid #f3f3f3;
+      transform: scale($img-scale);
+      padding: 12px 0;
+      margin: 15px 2px 29px;
+    }
+  }
+  &__card-header {
+    font-weight: 500;
+    line-height: 15px;
+    font-size: 12px;
+    margin: 0 auto 6px;
+  }
+  &__code {
+    padding-bottom: 15px;
+  }
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 20px;
+  }
+  &--wrap {
+    margin-bottom: 50px;
+  }
```

Comment:
use mix class for card instead of modifier
https://www.youtube.com/watch?v=JOJ-mENrg_s